### ISSUE TITLE
[Fishing] Terokkar Forest Empty Loot

### ISSUE THREAD

**Issue by Azalen** [Atlantiss/NetherwingBugtracker#4533]
**Current behaviour**:
While Fishing in Terrokar Forest I get "Empty Loot" ~75% of all the catches. This issue is not present in Zangarmarsh or Nagrand. On a sidenote, Empty Loot can still generate a skillup.

![wowscrnshot_012419_081159](https://user-images.githubusercontent.com/20336567/51661077-00d25400-1fb0-11e9-94e0-84f22f9ec0c4.jpg)

**Expected behaviour**: 
No Empty Loot.
As a ressource: http://old.elsanglinreborn.com/fish/terokkar_forest.html

**Server Revision**: 
[2270]

**Comment by Azalen** [comment]
Possible scenario: I don't have fishing 450 yet to be allowed to fish in open water in Terokkar Forest without getting Junk. Maybe "Empty Loot" represents Junk but is not correctly assigned to Junk Items. In any case, I will report if this issue is gone once I reach 450 fishing skill.

**Comment by wtRCH** [comment]
@Shanghi

**Comment by Kidtoucher** [comment]
this is still happening, its kind of major. please look into it.

**Comment by whitestar87** [comment]
This is still happening.  Darters have a disproportionately high cost due to empty loot tables when compared with other lvl 70 buff food.

**Comment by Nythius** [comment]
This is still happening

**Comment by Noadiah-Netherwing** [comment]
The animation appears and my fishing skill increases, however the loot box for the fish is empty. I tried turning on auto-loot, but this made no difference. I am using a +20 fishing pole and a +100 Lure (current fishing skill 362). I've tried disabling all my addons, deleted the cache folder, used a different fishing pole, and restarting the game.
![WoWScrnShot_032519_223946](https://user-images.githubusercontent.com/48968367/55036710-d02f8a80-4ff1-11e9-96a4-a04155211fea.jpg)
